spec/ddtrace/opentracer/span_spec.rb

```diff
@@ -47,23 +61,30 @@
       let(:timestamp) { Time.now }
       let(:fields) { { time_started: Time.now, account_id: '1234' } }
 
-      before(:each) do
+      # Expect a deprecation warning to be output.
+      it do
```

Comment:
Would ```it 'expects a deprecation warning to be output.'``` work here?

Reply:
You could do that, and that would be okay. What matters is that the test is easy to read and understand. The language as its written reads as "it expects log to output 'Span#log is deprecated.' to stderr", which I think is an abundantly clear statement, that does not require clarification by naming the example.

Reply:
Not to bikesched too much. I have mixed feelings about using nameless examples. Having explicit name to a example most often than not makes it more clearer than the code itself.

And while 1 line and relatively short  nameless examples are fine and easy to read most of the time. 
However examples like:
```
it do
 expect {x}.to change {z}
end
```
look like a exception from this rule. 

However since beauty lies in the eye of beholder I'm fine with both :). 

Lets come back ot this discussion a bit later when working on Rubocop rules

Reply:
This all comes down to style ultimately I think. Sometimes it's worth discussing the possible merits, and it's totally reasonable to suggest alternatives, but given that both suggested styles are clear and understandable, it doesn't seem like there's much to gain by dwelling too much on things like this. The same would apply if the circumstances were reversed, and it was the named example in the PR instead of the nameless one.

For what its worth though, the style choice here isn't arbitrary, it's an implementation of the http://www.betterspecs.org/ standard, which I think does a better job of explaining the reasoning than I can here.